activerecord/lib/active_record/attribute_methods/serialization.rb

```diff
@@ -15,6 +15,10 @@ def initialize(name, type)
         end
       end
 
+      included do
+        class_attribute :default_column_serializer, instance_accessor: false, default: Coders::YAMLColumn
```

Comment:
This does not create a global default but instead is configurable for each subclass of `ActiveRecord::Base`. The ability to configure each `serialize` is enough, plus a global default for compatibility should be enough.

Reply:
I can make it a global, but I thought having it inheritable made sense for say engines so that they can have their base abstract model, and define the serializer they use across the engine centrally.

Reply:
Hum, they could also redefine `ApplicationController.serialize` to provide defaults. 🤔

Reply:
And break if we change the signature. I prefer a class attribute.